package.json

```diff
@@ -18,10 +18,11 @@
   },
   "scripts": {
     "clean": "rm -rf dist",
-    "dev": "NODE_ENV=development ts-node-dev --respawn src/server.ts",
-    "build": "yarn clean && yarn tsc",
-    "start": "node dist/server.js",
-    "lint": "eslint . --ext .ts"
+    "dev": "NODE_ENV=development ts-node-dev --respawn src/main.ts",
```

Comment:
Not related to this file, but maybe it would be good to run tests pre-push with husky? This way if you forget to run the tests yourself you don't have to wait for CI to run

Reply:
When we used that in Atlas, I found that to be mostly annoying and Orion tests can take even a little longer to run. I'd skip that, as I believe situations in which we break tests unknowingly will be pretty rare, so it's okay to get that result from the CI. You can always run tests manually as well